tasks/utils.py

```diff
@@ -34,6 +35,11 @@
     RTLOADER_LIB_NAME = "libdatadog-agent-rtloader.so"
 RTLOADER_HEADER_NAME = "datadog_agent_rtloader.h"
 AGENT_VERSION_CACHE_NAME = "agent-version.cache"
+BUNDLED_AGENTS = {
+    # system-probe requires a working compilation environment for eBPF so we do not
+    # enable it by default but we enable it in the released artifacts.
+    AgentFlavor.base: ["process-agent", "trace-agent", "security-agent"],
+}
```

Comment:
I'm not a python expert, but I'm wondering if we need to use `AgentFlavor.base.name` as the key. The default value for flavor in `agent.py` is `AgentFlavor.base.name`

https://github.com/DataDog/datadog-agent/pull/19280/files#diff-738b385824d0f3488e27223c0a214bc564af28293d261af42eb65dd5353af3aeR107

If we use `AgentFloavor.base` I think in `agent.py` we are going to get always the default value. 

```py
import enum


# TODO: use StrEnum when we move to Python 3.11
class AgentFlavor(enum.Enum):
    base = 1
    iot = 2
    heroku = 3
    dogstatsd = 4

    def is_iot(self):
        return self == type(self).iot
        
BUNDLED_AGENTS = {
    AgentFlavor.base: ["process-agent", "trace-agent", "security-agent"],
}


print(BUNDLED_AGENTS.get(AgentFlavor.base.name, []))
# []

```

Reply:
I have no idea neither but using `AgentFlavor.base.name` also works so happy to make the change

Reply:
I missed the line you mentioned 🤦 
> So I suggest keeping it this way. Does it work for you?
Absolutely. Thanks for explaining it